third_party/testcases_new.h

```diff
@@ -0,0 +1,121 @@
+// Copyright(c) Microsoft Corporation.
```

Comment:
Can we rename this file to "nvbandwidth_testcases_new.h"? Then in Makefile, we can execute `cp ../nvbandwidth_testcases_new.h ./testcases_new.h`.